src/main/java/org/alliancegenome/curation_api/interfaces/crud/ConditionRelationCrudInterface.java

```diff
@@ -23,9 +27,19 @@ public interface ConditionRelationCrudInterface extends BaseIdCrudInterface<Cond
     @Path("/")
     @JsonView(View.FieldsAndLists.class)
     public ObjectResponse<ConditionRelation> create(ConditionRelation entity);
-    
+
     @POST
     @Path("/validate")
     @JsonView(View.FieldsAndLists.class)
     public ObjectResponse<ConditionRelation> validate(ConditionRelation entity);
+
+	@POST
+	@Path("/handles/search")
+	@Tag(name = "Database Search Endpoints")
+	@JsonView(View.FieldsAndLists.class)
+	public SearchResponse<ConditionRelation> findHandles(
+		@DefaultValue("0") @QueryParam("page") Integer page,
+		@DefaultValue("10") @QueryParam("limit") Integer limit,
+		@RequestBody HashMap<String, Object> params);
+
```

Comment:
Does this methed need to exist? We already have a `/search` method that has the same definition?

Reply:
Sounds reasonable. I created another key in the params hashmap that contains the nonNullFields.